js/nodes.js

```diff
@@ -37,9 +37,9 @@
     constructor(x, y) {
 
         if(x != undefined)
-            this.x = 1210
+            this.x = document.getElementById('btn-create').offsetLeft
```

Comment:
oh i see. how abt this?

Reply:
If that properly lines things up, that works for me.

Reply:
okay i will push it to main but if you find that it breaks anything feel free to put the variables back to 0